platform/users/adapters.py

```diff
@@ -0,0 +1,16 @@
+from typing import Any
+
+from allauth.account.adapter import DefaultAccountAdapter
+from allauth.socialaccount.adapter import DefaultSocialAccountAdapter
+from django.conf import settings
+from django.http import HttpRequest
+
+
+class AccountAdapter(DefaultAccountAdapter):
+    def is_open_for_signup(self, request: HttpRequest):
```

Comment:
We probably don't need a feature to block signups. Let's focus this pull request on Django 3.2, actually Django 4 now, compatibility :-)